### ISSUE TITLE
Extract metadata at the pixel level for each composite image

### ISSUE THREAD

**Issue by crangelsmith** [urbangrammarai/gee_pipeline#9]
We want to know the number of images in the composite per pixel, this is probably done with something like: https://developers.google.com/earth-engine/apidocs/ee-imagecollection-count

**Comment by crangelsmith** [comment]
This is the kind of data we can extract from GEE, this is the sum of counts of entries by pixel for each RBG band. This is for a 3 month composite. 

![image](https://user-images.githubusercontent.com/11162074/179532749-3d70ef3d-c7f8-4915-8d85-8e3adbeffa22.png)

**Comment by crangelsmith** [comment]
More results about this can be found in [this slides](https://thealanturininstitute.sharepoint.com/:p:/s/UrbanGrammar/EQGNmPw2tUZIrfQBFZOethABDq14RJ4zqHLh8Tibzs-kyA?e=1Ky5mt)

**Comment by crangelsmith** [comment]
After our conversation yesterday @andrewphilipsmith I took a look at the counts of each band independently and they all seem to have the same amount of counts, so as you mention, sum them over seems to be duplicating the information. We should probably just keep the counts of one band? 

![image](https://user-images.githubusercontent.com/11162074/180980934-2a2fdeda-a597-4243-994e-0ae5968f859c.png)

**Comment by crangelsmith** [comment]
After the meeting on the 26th of July a couple more points to investigate were raised by @andrewphilipsmith and @darribas. I added more details to [these slides](https://thealanturininstitute.sharepoint.com/:p:/s/UrbanGrammar/EQGNmPw2tUZIrfQBFZOethABDq14RJ4zqHLh8Tibzs-kyA?e=1Ky5mt) but here are the main points:

1. Reasons behind the "strip" of lower counts observed in London and Liverpool near the ocean. We decided to check similar figures in more central regions such as Leicester or Birmingham, the results seem to confirm our hypothesis:

![image](https://user-images.githubusercontent.com/11162074/181038144-e086209e-331c-40aa-8ecd-aed06bf870db.png)

2. Effect on total number of counts is only weather dependent or is there also a difference in the number of available images to query from gee. I fetched the total number of images before any cloud filtering for different years but same month period, and it seems to vary slightly but be around the ~70 images. Reasons for this variation are not clear to me yet.  

![image](https://user-images.githubusercontent.com/11162074/181038550-cc214a5b-6277-4255-8f0c-811abb88c03a.png)
